account_analytic_top_parent/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,29 @@
+For each analytic account, defines the top parent analytic account. If there is no parent account, the top parent account is the account itself.
+
+This module is useful for analytic reporting (via bi_view_editor for instance) of accounts organized in hierarchy. 
+
+The parent account cannot be used for reporting if there is more or less than two hierarchical layers, and if the parent accounts are used directly as account in the analytic lines. 
+
+Having top parent analytic account enable aggregate reporting for hierarchy of accounts that is not 
+
+For instance, let's assume we have a analytic account hierarchy like:
+
+- Account A / Account AB / Account ABC
+- Account B
+- Account C / Account CB
+
+If we group by parent analytic account we get:
+
+- Account A 
+- Account A / Account AB 
+- Account C / Account CB
+- no value / Account B
```

Comment:
Wouldn't it be :

- Account A
- Account A / Account AB
- False
- Account C

Reply:
yes correct